src/sage/symbolic/expression.pyx

```diff
@@ -10617,22 +10617,26 @@ cdef class Expression(Expression_abc):
         else:
             return self
 
-    def simplify(self, algorithm='maxima'):
+    def simplify(self, algorithm='maxima', **kwds):
         """
         Return a simplified version of this symbolic expression.
 
-        .. NOTE::
+        INPUT:
 
-           When using ``algorithm='maxima'``, this just sends the expression to Maxima
-           and converts it back to Sage. When ``algorithm='sympy'``,
-           sympy's simplify method is used.
+        - ``algorithm`` - one of :
 
-        INPUT:
+            - ``maxima`` : currently, sends the expression to
+              ``maxima`` and converts it back to Sage.
```

Comment:
```suggestion
            - ``maxima`` : currently, sends the expression to
              ``maxima`` and converts it back to Sage
```
and similar below to match our general style guide of inputs not having full-stops at the end.